src/components/ChooseOfferForm/ChooseOfferForm.tsx

```diff
@@ -172,9 +172,9 @@ const ChooseOfferForm: React.FC<Props> = ({
           </label>
         </div>
       )}
-      <div className={styles.offers}>{offers.map(renderOfferBox)}</div>
+      {<div className={styles.offers}>{!offers.length ? <p>{t('choose_offer.no_pricing_available')}</p> : offers.map(renderOfferBox)}</div>}
```

Comment:
Does it make sense to show "This Video" tab in case TVOD is not available?

Reply:
This video won't be shown if TVOD is not available..but if there are no prices I think it makes sense to show that there are no prices even on subscription asset.